examples/snapshot/test/specs/test.e2e.ts

```diff
@@ -0,0 +1,15 @@
+import { expect } from '@wdio/globals'
+
+describe('My Login application', () => {
+    it('should login with valid credentials', async () => {
+        await browser.url(`https://the-internet.herokuapp.com/login`)
+
+        await $('#username').setValue('tomsmith')
+        await $('#password').setValue('SuperSecretPassword!')
+        await $('button[type="submit"]').click()
+
+        await expect($('#flash')).toMatchSnapshot()
```

Comment:
Snapshotting the element object is not a valid usecase since `sessionId` and `elementId` are always different

Reply:
actually I intense to test the wrapped await behavior here :)

Reply:
updated